### PR TITLE
Why are you green?

### PR DESCRIPTION
**Pull request by samcunliffe**
Explains why the green tools are chosen for pages where there are greens chosen over some other amber/red tools. Doesn't explain the all-green or all-amber tool lists. Solves #37.

Please feel encouraged to suggest rewording etc, or even directly push to this branch.

### CODE REVIEW

**Review by paddyroddy — APPROVED** [review]
👌

File: pages/testing.md [hunk 1]
```diff
@@ -13,6 +13,11 @@ layout: default
 | [tox](https://tox.wiki/en/latest/index.html) | A framework that allows running tests and packaging in different environments. | 🟢 |
 | [unittest](https://docs.python.org/dev/library/unittest.html#module-unittest) | A framework built in to Python for writing and running tests. | 🟠 |
 
+<details>
+<summary>🟢 explanation</summary>
+We recommend `pytest` over `unittest` because of the extensive plugins and widespread use.
```
Comment (p-j-smith) [inline pages/testing.md:18]: maybe also refer to the cleaner style of pytest tests:
> We reocmmend `pytest` over `unittest` because of the extensive plugins, widespread use, and less boilerplate needed for writing tests.

File: pages/ci.md [hunk 2]
```diff
@@ -12,6 +12,10 @@ layout: default
 | [AppVeyor](https://www.appveyor.com/docs/) | Continuous integration and continuous delivery platform. | 🟠 |
 | [Bamboo](https://confluence.atlassian.com/bamboo/bamboo-documentation-289276551.html) | Atlassian continuous integration and continuous delivery platform. | 🟠 |
 
+<details>
+<summary> 🟢 explanation</summary>
+We have many projects using github CI, it's both convenient and works well. It's become the standard.
```
Comment (p-j-smith) [inline pages/ci.md:17]: Could also mention it's free for public repositories

**Comment by p-j-smith** [comment]
sorry just realised it's merged! ignore my comments

File: pages/testing.md (hunk 1 above)
Comment (samcunliffe) [reply pages/testing.md:18]: Merged too fast!
```suggestion
We recommend `pytest` over `unittest` because `pytest` tends to encourage a cleaner style, there are also extensive plugins and it's in widespread use.
```

File: pages/ci.md (hunk 2 above)
Comment (samcunliffe) [reply pages/ci.md:17]:
```suggestion
We have many projects using github CI, it's both convenient and works well. It's become the standard. Also, note that it's free for public repos.
```

File: pages/testing.md (hunk 1 above)
Comment (p-j-smith) [reply pages/testing.md:18]: no worries! I can't commit your suggestion so I'll make a separate pr
